openslides/agenda/static/js/agenda/agenda.js

```diff
@@ -153,6 +153,50 @@ angular.module('OpenSlidesApp.agenda', [])
       };
 })
 
-.controller('AgendaCSVImportCtrl', function($scope, Agenda) {
-    // TODO
+.controller('AgendaImportCtrl', function($scope, $state, Agenda) {
+    // import from textarea
+    $scope.importByLine = function () {
+        $scope.items = $scope.itemlist[0].split("\n");
+        $scope.importcounter = 0;
+        $scope.items.forEach(function(title) {
+            var item = {title: title};
+            item.weight = 0;  // TODO: the rest_api should do this
+            item.tags = [];   // TODO: the rest_api should do this
+            // TODO: create all items in bulk mode
+            Agenda.create(item).then(
+                function(success) {
+                    $scope.importcounter++;
+                }
+            );
+        });
+    }
```

Comment:
Ok. But this means, that when a file with 1000 Entries are imported where 50 Entries are invalid, there are 950 Entries imported and the user has to find the 50 invalid ones. That could be hard.